### DISCUSSION TITLE
Proposal: Build Try to everything using nix

### DISCUSSION THREAD

**Discussion by eveeifyeve** [Eveeifyeve/Eveeifyeve#2] [Ideas]
I want to make a proposal for all of my builds in my projects (`TeaClient` is not in this case! as it has it's own management), but for projects like `minecraft-essentials`, `osx-vim` and etc, I want to make everything try to build using nix like example building binarys, building docker images and more.

But I want to hear your feedback on this, like is this a good idea or not? Let me know your opinions as this is a proposal.

**Comment by nothendev** [comment]
binaries using nix are bad cus their rpaths and other stuff will refer to /nix/store/..stuff and not for example /lib/..stuff

docker images are dependent on binaries so same applies

**Reply by eveeifyeve** [reply]
> binaries using nix are bad cus their rpaths and other stuff will refer to /nix/store/..stuff and not for example /lib/..stuff
> 
> docker images are dependent on binaries so same applies

Yes, but for example nix is really great at being a deployment model and gives you the confidence "it works on my machine!" rather than binarys not working on certain architectures.

**Reply by eveeifyeve** [reply]
> binaries using nix are bad cus their rpaths and other stuff will refer to /nix/store/..stuff and not for example /lib/..stuff
> 
> docker images are dependent on binaries so same applies
But yeah you can also say no to nix store stuff in nix.
